### PR TITLE
feat: Update launch

### PR DESCRIPTION
**Pull request by mohandast52**
## Proposed changes

- Launch page updated
- A few components were created and 1 bug fix in the `services/contribute` page

## Screenshots/Recordings

https://github.com/user-attachments/assets/e2d814e8-1ea4-44ed-86d3-85b2eb73492a

## Things to keep in mind

- [ ] I confirm I have updated the meta title and description for the page

### CODE REVIEW

File: components/LaunchPage/BringAiAgentsEconomies.jsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        alt="predict"
+        width={1180}
+        height={348}
+        className="mx-auto"
+      />
+    </div>
+
+    <div className="text-center text-slate-500">
+      <Link
+        href="https://olas.network/agent-economies/predict"
```
Comment (Tanya-atatakai) [inline components/LaunchPage/BringAiAgentsEconomies.jsx:37]: can just be `agent-economies/predict`?

File: components/LaunchPage/HearFromHappyLaunchers.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    on Gnosis been executed by agents!
+  </p>
+);
+
+const TweetFooter = () => (
+  <div className="flex justify-between items-center">
+    <div className="text-gray-400">
+      10:16 AM · Nov 15, 2023 ·
+      {' '}
+      <span className="text-black font-semibold">7,268</span>
```
Comment (Tanya-atatakai) [inline components/LaunchPage/HearFromHappyLaunchers.jsx:73]: hmm shouldn't we update the number somehow? 🤔

File: components/LaunchPage/HearFromHappyLaunchers.jsx [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+);
+
+const TweetBody = () => (
+  <p>
+    Another big user has been AI agents by
+    {' '}
+    <ExternalLink href="https://x.com/autonolas">@autonolas</ExternalLink>
+    {' - '}
+    we have this ongoing challenge where agents gather information and trade
+    information and trade on prediction markets - this has lead to the majority
```
Comment (Tanya-atatakai) [inline components/LaunchPage/HearFromHappyLaunchers.jsx:58]: the text is a bit different from the original tweet, is it expected? the tweet says `... gather information and trade on prediction markets..` where we have `... gather information and trade information and trade..`

**Review by oaksprout — DISMISSED** [review]
lgtm

**Review by DavidMinarsch — DISMISSED** [review]

**Review by Tanya-atatakai — APPROVED** [review]

**Pull request merged**